packages/host/app/components/ai-assistant/panel.gts

```diff
@@ -51,31 +53,6 @@ interface Signature {
 }
 
 export default class AiAssistantPanel extends Component<Signature> {
-  get hasOtherActiveSessions() {
-    let oneMinuteAgo = new Date(Date.now() - 60 * 1000).getTime();
-
-    return this.aiAssistantPanelService.aiSessionRooms
-      .filter((session) => session.roomId !== this.roomResource?.roomId)
-      .some((session) => {
-        let isSessionActive = false;
-        isSessionActive =
-          this.matrixService.getLastActiveTimestamp(
-            session.roomId,
-            session.lastActiveTimestamp,
-          ) > oneMinuteAgo;
-
-        let lastMessageEventId = session.lastMessage?.eventId;
-
-        let hasSeenLastMessage = lastMessageEventId
-          ? this.matrixService.currentUserEventReadReceipts.has(
-              lastMessageEventId,
-            )
-          : false;
-
-        return isSessionActive && !hasSeenLastMessage;
-      });
-  }
-
```

Comment:
This is just moved to where the rest of the component Javascript lives, beneath the template.